### ISSUE TITLE
Binding error on keydown/up events

### ISSUE THREAD

**Issue by aleph-naught2tog** [phoenixframework/phoenix_live_view#115]
### Environment

* Elixir version (elixir -v): Elixir 1.8.1 (compiled with Erlang/OTP 21)
* Phoenix version (mix deps): phoenix 1.4.2
* NodeJS version (node -v): v11.10.1
* NPM version (npm -v): 6.7.0
* Operating system: Mac OS 10.14.3

### Actual behavior

On keydown, there is a Javascript error, and it never makes it to the server. (I'm currently handling _no_ events in my live view, so it definitely isn't being handled.)

Reproducible [here](https://github.com/aleph-naught2tog/phx_live_event_samples/tree/working-button-version-bug). The repo's for a mini tutorial I was/am working on, so is about as trivial as it could be, happily! (Reproducible in `master` too, I just did a separate branch to add the button contrast.)

I've checked it for `phx-keyup` and `phx-keydown` as well; neither will fire. 

`phx-click` _does_ fire to the server, where it throws an expected error.

`phx-keydown` with _no_ `phx-target` will throw the JS error; `phx-keydown` _with_ a `phx-target="document"` will not throw an error, nor will it send an event to the server. (I've got those in the repo since that's pretty unclear written out)

```
// in the JS console
phoenix_live_view.js:1493 Uncaught ReferenceError: el is not defined
    at eval (phoenix_live_view.js:1493)
    at e.value (phoenix_live_view.js:1427)
    at eval (phoenix_live_view.js:1492)
(anonymous) @ phoenix_live_view.js:1493
value @ phoenix_live_view.js:1427
(anonymous) @ phoenix_live_view.js:1492
```

### Expected behavior

I expect the keydown event to fire, and to get an error in the server log about not handling the event in the live view.

**Comment by aleph-naught2tog** [comment]
If I manually swap out my JS file for [the one from here](https://github.com/phoenixframework/phoenix_live_view/commit/12747538054eeb29728d6334dba1e9f44b1cbaa7), then the app behaves as expected.

**Comment by zakora** [comment]
Same kind of issue for me:
- if `target` is neither `document` nor `window` then no key event happens
- if `target` is either `document` or `window` then for a given key, only the first occurence will be received by the server.

For example, by running the Snake demo from [phoenix_live_view_example](https://github.com/chrismccord/phoenix_live_view_example) and applying the following patch, we can see that when using several times the down arrow, then several times the up arrow the server will only display `ArrowDown` once and `ArrowUp` once:

```
[info] GET /snake
[debug] Processing with DemoWeb.PageController.snake/2
  Parameters: %{}
  Pipelines: [:browser]
[info] Sent 200 in 2ms
[info] CONNECT Phoenix.LiveView.Socket
  Transport: :websocket
  Connect Info: %{}
  Parameters: %{"vsn" => "2.0.0"}
[info] Replied Phoenix.LiveView.Socket :ok
[info] Replied phoenix:live_reload :ok
"ArrowDown"
"ArrowUp"
```

the patch:
```patch
diff --git a/lib/demo_web/live/snake_live.ex b/lib/demo_web/live/snake_live.ex
index 3324438..da38171 100644
--- a/lib/demo_web/live/snake_live.ex
+++ b/lib/demo_web/live/snake_live.ex
@@ -178,11 +178,10 @@ defmodule DemoWeb.SnakeLive do
     |> compact_tail()
   end

-  defp turn(socket, "ArrowLeft"), do: go(socket, :left)
-  defp turn(socket, "ArrowDown"), do: go(socket, :down)
-  defp turn(socket, "ArrowUp"), do: go(socket, :up)
-  defp turn(socket, "ArrowRight"), do: go(socket, :right)
-  defp turn(socket, _), do: socket
+  defp turn(socket, val) do
+    IO.inspect val
+    socket
+  end

   defp go(socket, heading) do
     update(socket, :pending_headings, fn
```

**Comment by aleph-naught2tog** [comment]
Hopefully I didn't overstep but I believe I fixed it in #118 .

**Comment by chrismccord** [comment]
This has been fixed in 09da148f546ab69cb77106d3114936e4a2d7756f . Thank you for reporting and taking the time to deep dive a fix! In my event capture refactor, I updated the phx-key docs to:

```
The onkeydown and onkeyup events are supported via
the `phx-keydown`, and `phx-keyup` bindings. By
default, the bound element will be the event listener, but an
optional `phx-target` may be provided which may be `"window"`.
```

So we no longer target DOM ids, and we dropped `document`. So now `window` is the only target. I have verified the fix with these restrictions given your example repo. Thanks!
